velox/vector/BaseVector.cpp

```diff
@@ -338,6 +338,10 @@ VectorPtr BaseVector::create(
           true /*isSorted*/,
           0 /*representedBytes*/);
     }
+    case TypeKind::SHORT_DECIMAL:
+    case TypeKind::LONG_DECIMAL: {
+      return createEmpty<TypeKind::SHORT_DECIMAL>(size, pool, type);
```

Comment:
This should probably not be supported as there is no precision/scale?

Reply:
The `type` parameter in createEmpty<>() call would have precision and scale info.